aws-ecs/terraform/main.tf

```diff
@@ -0,0 +1,118 @@
+resource "random_id" "deployment" {
+  prefix      = "${local.name_prefix}-"
+  byte_length = 4
+}
+
+locals {
+  default_controller_settings = {
+    deploy                    = true
+    container_name            = "rookout-controller"
+    serer_mode                = "PLAIN"
+    dop_no_ssl_verify         = true
+    onprem_enabled            = true
+    certificate_bucket_prefix = null
+    certificate_bucket_name   = null
+    certificate_arn           = null
+    publish_lb                = false
+    task_cpu                  = 512
+    task_memory               = 1024
+    container_cpu             = 256
+    container_memory          = 512
+    container_port            = 7488
+    load_balancer_port        = 7488
+  }
+  default_datastore_settings = {
+    deploy                    = true
+    container_name            = "rookout-datastore"
+    serer_mode                = "PLAIN"
+    cors_all                  = true
+    in_memory_db              = true
+    certificate_bucket_prefix = null
+    certificate_bucket_name   = null
+    certificate_arn           = null
+    task_cpu                  = 512
+    task_memory               = 1024
+    container_cpu             = 256
+    container_memory          = 512
+    storage_size              = 20
+    container_port            = 8080
+    load_balancer_port        = 443
+  }
+  controller_settings = merge(local.default_controller_settings, var.controller_settings)

```

Comment:
the default values already declared in variables.tf. why declare it twice and merge?